protocol/message-header.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package protocol;
+
+message MessageHeader {
+
+    enum MessageType {
+        CONTROL = 0;
+        DATA = 1;
+    }
+
+    enum Service {
+        COMPUTER_VISION = 0;
+        MACHINE_LEARNING = 1;
+        GEOMETRY_PROCESSING = 2;
+        RENDERING = 3;
+        HEAD_TRACKING = 4;
+        IMAGE_PROVIDING = 5;
```

Comment:
What's image_providing again? Is that directions images from the phone or something?

Reply:
Thanks for merging without answering my question!

Reply:
This refers to Kinect server